### PR TITLE
pdksync - Remove Puppet 5 from testing and bump minimal version to 6.0.0

### PR DESCRIPTION
**Pull request by carabasdaniel**
Remove Puppet 5 from testing and bump minimal version to 6.0.0
pdk version: `1.19.0.pre (47)`

### CODE REVIEW

File: .github/workflows/auto_release.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      with:
+        args: 'release prep --force'
+      env:
+        CHANGELOG_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: "Get Version"
+      if: ${{ github.repository_owner == 'puppetlabs' }}
+      id: gv
+      run: |
+        echo "::set-output name=ver::$(cat metadata.json | jq .version | tr -d \")"
```
Comment (ekohl) [inline .github/workflows/auto_release.yml:49]: Probably needs to be fixed in pdk-templates, but there's a oneliner for this
```suggestion
        echo "::set-output name=ver::$(jq --raw-output .version metadata.json)"
```

File: .github/workflows/auto_release.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    - name: "Commit changes"
+      if: ${{ github.repository_owner == 'puppetlabs' }}
+      run: |
+        git config --local user.email "action@github.com"
```
Comment (ekohl) [inline .github/workflows/auto_release.yml:54]: Is it really a good idea to claim some email address `@github.com`? For anonymous commits they use `USER@users.noreply.github.com` so I'd suggest:
```suggestion
        git config --local user.email "${{ github.repository_owner }}@users.noreply.github.com"
```

File: .github/workflows/auto_release.yml (hunk 1 above)
Comment (pmcmaw) [reply .github/workflows/auto_release.yml:49]: This has been updated in the pdk-templates: 
https://github.com/puppetlabs/pdk-templates/pull/409

File: .github/workflows/auto_release.yml (hunk 2 above)
Comment (pmcmaw) [reply .github/workflows/auto_release.yml:54]: This has been updated in the pdk-templates: 
https://github.com/puppetlabs/pdk-templates/pull/409

**Pull request merged**